src/AspNet.Security.OAuth.SuperOffice/AspNet.Security.OAuth.SuperOffice.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>ASP.NET Core security middleware enabling SuperOffice authentication.</Description>
+    <Authors>Anthony Yates</Authors>
+    <PackageTags>superoffice;aspnetcore;authentication;oauth;security</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.4.0" />
```

Comment:
Dependency versions should be declared in [`Versions.props`](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dev/eng/Versions.props).